playground/Directory.Build.targets

```diff
@@ -10,7 +10,7 @@
     <SkipEventHubsEmulation Condition="'$(SkipEventHubsEmulation)' == '' and ('$(RepoRoot)' == '' or '$(ContinuousIntegrationBuild)' == 'true' or '$(CODESPACES)' == 'true')">true</SkipEventHubsEmulation>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(IsAspireHost)' == 'true' and '$(SkipDashboardProjectReference)' != 'true'">
+  <ItemGroup Condition="$(MSBuildProjectName.EndsWith('AppHost')) and '$(SkipDashboardProjectReference)' != 'true'">
```

Comment:
Can we make the `Aspire.Hosting.AppHost` package add something, e.g. property or project capability, to make conditions like a bit more concrete?

Reply:
We could, but note that this is only needed inside our repo. For Aspire consumers scenario, it is straight forward to determine whether or not their projects are AppHost projects as well as if they are using the SDK or not and which version. In the repo it's different, as we consume things directly as opposed to doing it via a package ref, which is what makes us have to write conditions like this.